logparser/logparser.py

```diff
@@ -156,11 +156,15 @@ def _parse_log(self):
                                               True)
 
         # While there is a new line, parse it.
-        line = True  # For the first condition.
-        while line:
+        line = ""
+        while line is not None:
```

Comment:
Because then `line == ""` will be false too and don't want to stop for empty lines.